### PR TITLE
Feature/remove old apr

### PR DESCRIPTION
**Pull request by Velua**

### CODE REVIEW

**Comment by Velua** [comment]
@pingustar are we no longer using this data?

**Comment by pingustar** [comment]
> @pingustar are we no longer using this data?

I am not sure I have any information regarding this PR. What is the issue # ?

**Comment by Velua** [comment]
There is none, I'm pretty sure the data coming out from get getter was never getting used by the view layer at the time so I was wondering if we could delete it but wanted to get your thoughts.

File: src/components/protection/ProtectedTable.vue [hunk 1]
```diff
@@ -533,8 +533,6 @@ export default class ProtectedTable extends BaseComponent {
         return row.fullyProtected.usdValue;
       case "protectedAmount":
         return row.protectedAmount.usdValue;
-      case "apr":
-        return row.apr.day;
```
Comment (pingustar) [inline src/components/protection/ProtectedTable.vue:537]: we need this for sorting by APR.

File: src/components/data/pooltokentables/TablePools.vue [hunk 2]
```diff
@@ -91,10 +91,6 @@ export default class TablePools extends BaseComponent {
     return new BigNumber(percentage).gte(0) ? formatPercent(percentage) : "N/A";
   }
 
-  get aprsExist() {
-    return this.items.some(pool => pool.apr);
-  }
-
```
Comment (pingustar) [inline src/components/data/pooltokentables/TablePools.vue:97]: this looks fine.

**Pull request closed without merging**